File: namsor/python/src/JapaneseNameMatchFeedbackLoopExample.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.JapaneseApi(api_client).japanese_name_match_feedback_loop(
            japanese_surname_latin="Tessai",
            japanese_given_name_latin="Tomioka",
            japanese_name="富岡 鉄斎",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling JapaneseApi#japanese_name_match_feedback_loop: %s\n" % e)

File: namsor/ruby/src/JapaneseNameMatchFeedbackLoopExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["X-API-KEY"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::JapaneseApi.new.japanese_name_match_feedback_loop(
        "Tessai", # japanese_surname_latin
        "Tomioka", # japanese_given_name_latin
        "富岡 鉄斎", # japanese_name
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling JapaneseApi#japanese_name_match_feedback_loop: #{e}"
end
